api/hooks/permissions/index.js

```diff
@@ -19,9 +19,18 @@ class Permissions extends Marlinspike {
     if (!_.isObject(sails.config.permissions)) sails.config.permissions = { }
 
     /**
-     * Local cache of Model name -> id mappings to avoid excessive database lookups.
+     * workaround for #219:
+     * If your _actually_ want to populate blueprint models, you should set `populateForce` filed in config
      */
-    this.sails.config.blueprints.populate = false
+    var populateForce = this.sails.config.blueprints.populateForce;
+    if (populateForce) {
+      this.sails.config.blueprints.populate = !!populateForce;
```

Comment:
I know about side-effects of using !! to coercion to boolean value.

But i think it's pretty legal to expect not only exactly true value but all "truish" like 1, {}, or 0, ""